Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/gallery.xhtml

```diff
@@ -19,6 +19,9 @@
         xmlns:a="http://xmlns.jcp.org/jsf/passthrough">
     <p:panel id="galleryWrapperPanel" styleClass="wrapperPanel">
 
+        <!--@elvariable id="mayWrite" type="boolean"-->
+        <ui:param name="mayWrite" value="#{SecurityAccessController.hasAuthorityToEditProcessImages()}"/>
```

Comment:
maybe call it
```
editProcessImage
editImageAuthority
```

The one most fitting the other params in the xhtml would be:
`editImages`